client/control/tests/reducers/test_columnReducer.js

```diff
@@ -0,0 +1,129 @@
+import appReducer from 'control/reducers';
+import * as actions from 'control/actions/ColumnActions';
+
+import {
+  initialState,
+} from 'control/tests/fixtures';
+
+/**
+ * Utility function to clone the initialState column array and its values
+ * (This prevents nested objects from being cloned by reference)
+ *
+ * @return {Array} New array with cloned initialState values
+ */
+const cloneInitialArray = () => JSON.parse(JSON.stringify(initialState.columns));
+
+
+/**
+ * Utility function to get the index of a column
+ * which is NOT shown by default.
+ *
+ * @return {Number} Index of non-default column to test against.
+ */
+const getNondefaultColumn = () => {
+  let foundIndex;
+  initialState.columns
+    .find((col, index) => {
+      foundIndex = index;
+      return !col.enabled;
+    });
+
+  if (!foundIndex) {
+    console.warn('Warning! No non-default columns to test against.');
+  }
+  return foundIndex;
+};
+
+/**
+ * Column reducer tests
+ */
+describe('Column reducer', () => {
+  it('should return initial state by default', () => {
+    expect(appReducer(undefined, {})).toEqual(initialState);
+  });
+
+  it('should have default columns to display', () => {
+    const defaults = [
+      'name',
+      'action',
+      'enabled',
+      'last_updated',
+      'metadata',
+    ];
+
+    // get the columns that are enabled,
+    // and their shorthand values
+    const foundDefaults = cloneInitialArray()
+      .filter(col => col.enabled)
+      .map(col => col.value);
+
+    // arrays of default enableds should match
+    expect(foundDefaults).toEqual(defaults);
+  });
+
+  it('should handle UPDATE_COLUMN (enable)', () => {
```

Comment:
I think this and the next test should be nested in a describe, like  you did for `LOAD_SAVED_COLUMNS`.